gtsam/slam/SmartFactorBase.h

```diff
@@ -37,12 +37,14 @@
 namespace gtsam {
 
 /**
- * @brief  Base class for smart factors
+ * @brief  Base class for smart factors.
  * This base class has no internal point, but it has a measurement, noise model
  * and an optional sensor pose.
- * This class mainly computes the derivatives and returns them as a variety of factors.
- * The methods take a Cameras argument, which should behave like PinholeCamera, and
- * the value of a point, which is kept in the base class.
+ * This class mainly computes the derivatives and returns them as a variety of
+ * factors. The methods take a CameraSet<CAMERA> argument and the value of a
+ * point, which is kept in the derived class.
+ *
+ * @tparam CAMERA should behave like a set of PinholeCamera.
```

Comment:
I don't understand the comment "CAMERA should behave like a set of PinholeCamera.". (why "a set of"?)
Btw: In PR #861 I made sure this works nicely with a SphericalCamera, so I hope this comment is/will be removed.

Reply:
It's wrong. I'll fix it.